GDevelop.js/__tests__/Core.js

```diff
@@ -758,41 +768,84 @@ describe('libGD.js', function () {
-      expect(grandChild.contains(grandChild, true)).toBe(false);
-      expect(grandChild.contains(grandChild, false)).toBe(false);
-      expect(grandChild.contains(child1, true)).toBe(false);
-      expect(grandChild.contains(child2, true)).toBe(false);
-      expect(grandChild.contains(parentVariable, true)).toBe(false);
-      expect(grandChild.contains(child1, false)).toBe(false);
-      expect(grandChild.contains(child2, false)).toBe(false);
-      expect(grandChild.contains(parentVariable, false)).toBe(false);
-
-      parentVariable.removeRecursively(grandChild);
+
+      expect(child1.contains(grandChild1, true)).toBe(true);
+      expect(child1.contains(grandChild1, false)).toBe(true);
+      expect(child2.contains(grandChild1, true)).toBe(false);
+      expect(child2.contains(grandChild1, false)).toBe(false);
+
+      expect(child1.contains(grandChild2, true)).toBe(false);
+      expect(child1.contains(grandChild2, false)).toBe(false);
+      expect(child2.contains(grandChild2, true)).toBe(true);
+      expect(child2.contains(grandChild2, false)).toBe(true);
+
+      expect(grandChild1.contains(grandChild1, true)).toBe(false);
+      expect(grandChild1.contains(grandChild1, false)).toBe(false);
+      expect(grandChild1.contains(child1, true)).toBe(false);
+      expect(grandChild1.contains(child2, true)).toBe(false);
+      expect(grandChild1.contains(parentVariable, true)).toBe(false);
+      expect(grandChild1.contains(child1, false)).toBe(false);
+      expect(grandChild1.contains(child2, false)).toBe(false);
+      expect(grandChild1.contains(parentVariable, false)).toBe(false);
+
+      expect(grandChild2.contains(grandChild1, true)).toBe(false);
+      expect(grandChild2.contains(grandChild1, false)).toBe(false);
+      expect(grandChild2.contains(child1, true)).toBe(false);
+      expect(grandChild2.contains(child2, true)).toBe(false);
+      expect(grandChild2.contains(parentVariable, true)).toBe(false);
+      expect(grandChild2.contains(child1, false)).toBe(false);
+      expect(grandChild2.contains(child2, false)).toBe(false);
+      expect(grandChild2.contains(parentVariable, false)).toBe(false);
+
+      parentVariable.removeRecursively(grandChild1);
       expect(child1.hasChild('GrandChild')).toBe(false);
       expect(child1.getChildrenCount()).toBe(0);
       expect(parentVariable.getChildrenCount()).toBe(2);
 
+      parentVariable.removeRecursively(grandChild2);
+      expect(child2.getAllChildrenArray().size()).toBe(0);
+      expect(child2.getChildrenCount()).toBe(0);
+      expect(parentVariable.getChildrenCount()).toBe(2);
+
       parentVariable.removeRecursively(child2);
       expect(parentVariable.getChildrenCount()).toBe(1);
       expect(parentVariable.hasChild('Child1')).toBe(true);
       expect(parentVariable.hasChild('Child2')).toBe(false);
 
+      grandChild1.delete();
+      grandChild2.delete();
+      child1.delete()
+      child2.delete()
```

Comment:
We should not delete grandChild1/2 and child2, because they were already destroyed by removeRecursively. This means that here you would delete an already deleted object (grandChild1/2 and child2 are just references to a variable, that was removed - and remember that in libGD.js things are not garbage collected like in JS, the lifetime of objects depends on where they are stored). 
I'm removing these and app veyor tests should pass :) 
Glad that app veyor found this! Unsure why this did not crash on Travis. The beauty of "undefined behavior"!